documentation/concepts/transports.md

```diff
@@ -1,3 +1,24 @@
 ```yaml
 title: Transports
 ```
+
+Machines, within IoT systems, communicate amongst each other by exchanging
+messages over a variety of transport protocols like TCP, UDP, Bluetooth,
+ZigBee, LoRaWAN etc.
+
+Ockam includes a set of secure communication protocols, like
+[secure channels](./secure_channels) that provide their functionality in a
+way that is agnostic to the underlying means of transporting messages.
+This is achieved using an abstract `Transport` interface.
+
+A concrete implementation of the `Transport` interface is called an Ockam
+Transport. Over time there will be many such implementations.
```

Comment:
Did we agree to call it an Ockam Transport? Or was the transport going to refer to the lower-level, encryption-agnostic layer with Ockam Channel be the secure encrypted channel?

Reply:
transport is the the low level simple abstraction